stdlib/3/concurrent/futures/_base.pyi

```diff
@@ -29,19 +26,19 @@ class Future(Generic[_T]):
     def running(self) -> bool: ...
     def done(self) -> bool: ...
     def add_done_callback(self, fn: Callable[[Future], Any]) -> None: ...
-    def result(self, timeout: float = ...) -> _T: ...
-    def exception(self, timeout: float = ...) -> Exception: ...
+    def result(self, timeout: float = None) -> _T: ...
```

Comment:
I'm sorry I wasn't clear, but could you make the type explicitly `Optional[int]` or `Optional[float]` everywhere you did this too? (And actually then the default could be `...`)